libethcore/SealEngine.cpp

```diff
@@ -51,6 +51,7 @@ void SealEngineFace::verifyTransaction(ImportRequirements::value _ir, Transactio
 SealEngineFace* SealEngineRegistrar::create(ChainOperationParams const& _params)
 {
 	SealEngineFace* ret = create(_params.sealEngineName);
+	assert(!!ret && "Seal engine not found.");
```

Comment:
yeah, might also work without `!!`. The `!!` is to force bool context, things like `assert(ret)` sometimes do not work on windows.